sag_ji4.par

```diff
@@ -1,306 +1,306 @@
+-39.276834602139000	(!(	44/45	a	-80
+-38.855258559897800	`(!(	1377/1408	`a	-79
+-38.293157170242900	``(!(	1664/1701	``a	-78.5
+-38.061940349778500	\!~	45/46		-78
+-37.871581128001700	.~!)	524288/535815		-77
+-37.309479738346700	'(!(	8019/8192	'a	-76
+-36.887903696105500	`'(!(	23552/24057	`'a	-75.5
+-36.588490003137900	,,~!)	47/48		-75
+-36.325802306450600	,~!)	327680/334611		-74
+-35.904226264209400	~!)	48/49		-73
+-35.342124874554400	`~!)	243/248		-72.5
+-35.118091464366500	.(!	49/50	.j	-72
+-34.920548832313200	`.(!	247/252	`.j	-71
+-34.358447442658300	,,(!	50/51	,,j	-70
+-33.936871400417100	,(!	51/52	,j	-69
+-33.374770010762100	(!	45056/45927	j	-68
+-32.953193968520900	`(!	368/375	`j	-67
+-32.391092578866000	,!/	22113/22528	,k	-66
+-31.969516536624800	!/	54/55	k	-65
+-31.407415146969900	'(!	55/56	'j	-64
+-30.985839104728600	)!)	56/57		-63
+-30.564263062487400	`)!)	2673/2720		-62
+-30.002161672832500	,.!)	3072/3125	,.t	-61
+-29.580585630591300	.!)	3584/3645	.t	-60
+-29.018484240936400	`.!)	2097152/2132325	`.t	-59
+-28.596908198695100	,,!)	212992/216513	,,t	-58
+-28.034806809040200	,!)	19683/20000	,t	-57
+-27.613230766799000	!)	63/64	t	-56
+-27.051129377144100	`!)	64/65	`t	-55
+-26.629553334902900	``!)	65/66	``t	-54
+-26.067451945247900	,'!)	2048/2079	,'t	-53
+-25.645875903006700	'!)	2066715/2097152	't	-52
+-25.083774513351800	`'!)	6561/6656	`'t	-51.5
+-24.885981585695500	)\!	40960/41553		-51
+-24.662198471110600	`)\!	3328/3375		-50
+-24.100097081455600	``)\!	1215/1232		-49.6
+-23.955448448755500	,.\!	1048576/1063125	,.\	-49
+-23.678521039214400	.\!	524288/531441	.\	-48
+-23.116419649559500	')\!	75/76		-47
+-22.694843607318300	,,\!	3159/3200	,,\	-46
+-22.132742217663300	,\!	864/875	,\	-45
+-21.711166175422100	\!	80/81	\	-44
+-21.289590133180900	`\!	65536/66339	`\	-43
+-20.727488743526000	``\!	253/256	``\	-42
+-20.305912701284800	)!~	19456/19683		-41
+-19.743811311629800	'\!	2025/2048	'\	-40
+-19.322235269388600	`'\!	90/91	`'\	-39
+-18.760133879733700	,,~~!	137781/139264		-38
+-18.338557837492500	,~~!	7936/8019		-37
+-17.776456447837500	~~!	98/99		-36
+-17.354880405596300	,!~	507/512	,z	-35
+-16.792779015941400	!~	729/736	z	-34
+-16.371202973700200	`!~	32768/33075	`z	-33
+-15.809101584045200	``!~	409600/413343	``z	-32
+-15.387525541804000	,~!(	567/572	,o	-31
+-14.825424152149100	~!(	4096/4131	o	-30
+-14.403848109907900	`~!(	243/245	`o	-29
+-13.841746720252900	``~!(	125/126	``o	-28
+-13.420170678011700	,'~!(	5589/5632	,'o	-27
+-12.998594635770500	'~!(	135/136	'o	-26
+-12.436493246115600	)~!	143/144		-25
+-12.014917203874400	.)!(	2816/2835	.i	-24
+-11.452815814219400	`.)!(	2083725/2097152	`.i	-23
+-11.031239771978200	,,)!(	106496/107163	,,i	-22
+-10.469138382323300	,)!(	2187/2200	,i	-21
+-10.047562340082100	)!(	891/896	i	-20
+-9.485460950427140	`)!(	131072/131769	`i	-19
+-9.063884908185930	~!	2176/2187	s	-18
+-8.501783518530990	`~!	1024/1029	`s	-17
+-8.080207476289780	.!(	224/225	.n	-16
+-7.518106086634840	`.!(	19600/19683	`.n	-15

```

Comment:
seems like we moved from an 11²k to a 5²7²k on this one?

Reply:
Ah yes. I vaguely remember, in my 2am brain fog, thinking "Too hard. I'll have to come back to that one". Then I promptly forgot about it. :-)

That's hard to understand. 121k has a SoPF>3 of 22 and a 3-exp of 5, while 1225k has a SoPF>3 of 24 and a 3-exp of 9. So 1225k looks like the winner to me. I don't remember anything about this one.

I searched email for 121k and 1225k and only found one message, which is already up here:
http://forum.sagittal.org/viewtopic.php?f=4&t=448&p=1373&hilit=121k+1225k#nondistinguished
It confirms that the primary comma is 1225k.

But 121k has twice as many deemed occurrences, in the spreadsheet here:
http://forum.sagittal.org/viewtopic.php?p=1639#p1639

It is also 1225k in George's JI Notation Spreadsheet. 

George's weighted complexity shows 121k as less complex.
http://forum.sagittal.org/viewtopic.php?p=1650#p1650

So every metric seems to be saying it should be 121k, not 1225k.

Is 1225k the SoFLS of ,'|( for some secondary comma of ,| ?

Reply:
> Ah yes. I vaguely remember, in my 2am brain fog, thinking "Too hard. I'll have to come back to that one". Then I promptly forgot about it. :-)

Happens to the best of us.

> Is 1225k the SoFLS of ,'|( for some secondary comma of ,| ?

Yes. The difference between '|(, the 25/7k, and the 1/121k is 3024/3025, or the 7/5²11²n, which is about -0.5724 cents, so that's what ,! would represent were the primary comma for ,'|( to be set (back?) to the 1/121k. That value is solidly inside the secondary comma zone for ,! which ranges between 0.211 and 0.773 cents. I agree with your reasoning above that the 1/121k should be preferable to the 1/1225k. 

The only evidence we have against it its empirical existence in the current materials (it appears to be one of those missing from SagittalJI.gif). That and its higher limit (11 vs the 7 of the 1/1225k). 

What do you think? Should I go ahead and change it in the JI Notation Spreadsheet?

Reply:
Not yet. I think this discussion needs to be moved to the forum, so others have a chance to comment.

Reply:
Yes, I considered suggesting that myself. I’ll do it soon. For now I’ll standardize with the other materials. This is the last conversation so I’ll go ahead and merge this pass. ASAP I will make all the requested changes.

Reply:
Yes, I considered suggesting that myself. I’ll do it soon. For now I’ll standardize with the other materials. This is the last conversation so I’ll go ahead and merge this pass. ASAP I will make all the requested changes.